src/components/calcite-tree/calcite-tree.tsx

```diff
@@ -44,6 +44,7 @@ export class CalciteTree {
   @Prop({ reflect: true }) theme: Theme;
 
   /** @internal If this tree is nested within another tree, set to false */
+  // eslint-disable-next-line @esri/calcite-components/strict-boolean-attributes
   @Prop({ reflect: true, mutable: true }) root = true;
```

Comment:
Do we have an issue to clean this up? Even though it's internal :)

Reply:
I'll create one.

Reply:
https://github.com/Esri/calcite-components/issues/2164